src/components/App/App.tsx

```diff
@@ -1,21 +1,137 @@
-import React from 'react';
-import { withTranslation, WithTranslation } from 'react-i18next';
-import s from './App.scss';
+import React, { Fragment } from 'react';
+import {
+  Layout,
+  Cell,
+  Page,
+  Breadcrumbs,
+  Button,
+  Card,
+  FormField,
+  Input,
+  Heading,
+  Dropdown,
+  AddItem,
+  IconButton,
+  Text,
+  Box,
+} from 'wix-style-react';
+import DeleteSmall from 'wix-ui-icons-common/DeleteSmall';
 
-interface AppProps extends WithTranslation {}
-
-class App extends React.Component<AppProps> {
-  render() {
-    const { t } = this.props;
-
-    return (
-      <div className={s.root}>
-        <h2 className={s.title} data-hook="app-title">
-          {t('app.title', { who: 'Yoshi' })}
-        </h2>
-      </div>
-    );
-  }
-}
-
-export default withTranslation()(App);
+export default () => (
+  <Fragment>
```

Comment:
We should wrap the application with the component `<WixStyleReactProvider/>` to have the access of new features / experiments as explained [here](https://github.com/wix-private/crash-course-wsr-form#exercise-1---create-a-page). You can check out its documentation [here](https://www.wix-style-react.com/?path=/story/components-api-components--wixstylereactprovider).